pkg/template/processor.go

```diff
@@ -110,6 +111,10 @@ func processOnNode(resultChan chan error, ip, cmd, expectedValue string) {
 	cmds := strings.Split(cmd, ",")
 	for _, c := range cmds {
 		err = assert.ValidateOnNode(
+			assert.AsyncOpt{
```

Comment:
also this implementation is now calling objects from various dependencies, which can cause confusion. For ex:
- AsyncOpt is called from assert
- Timeout is a value from shared.PointerDuration
- Ticker is from time
Previous implementation was simpler, just using time library

Reply:
yeah, but previous implementation was hard coded, and we are using pkgs, not a single source, so we need to call stuff from their respectively place